NExtends/Expressions/ExpressionEqualityComparer.cs

```diff
@@ -0,0 +1,239 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NExtends.Expressions
+{
+	/// <summary>
+	/// source : http://stackoverflow.com/questions/283537/most-efficient-way-to-test-equality-of-lambda-expressions
+	/// </summary>
+	public static class ExpressionEqualityComparer
+	{
+		public static bool Eq(
+			LambdaExpression x,
+			LambdaExpression y)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Consider using more general type 'System.Linq.Expressions.Expression' instead of 'System.Linq.Expressions.LambdaExpression'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS3242)